### ISSUE TITLE
Configurable repository name

### ISSUE THREAD

**Issue by am93** [isacikgoz/gitbatch#69]
I have multiple repositories with same names in different projects, but of course in different folders.

For example:
-> FOLDER1
--> repoName1
--> repoName2
-> FOLDER2
--> repoName3
--> repoName1

In app I now see:

-- repoName1   
-- repoName2
-- repoName3
-- repoName1

I have a problem, because I dont know which repoName1 belongs to which folder. It would be awesome if there would be some configuration available, where you could define how repository is displayed in app - for example: instead of just name also name of parent directory.

With this addition it would be perfect app for me (it is already awesome right now) 😄 .

**Comment by isacikgoz** [comment]
Hey @am93 thanks for the feedback. 

That can be possible, but we should decide on a convenient solution. There can be a few options:

- Only display parent directory if the repo name has the same
- Show all repos parent (if configured)
- Show absolute file path when pressed on a button

The most easy one is the last, but not so user friendly. Second option on the other hand, it is too much detail to put this on a app configuration. First one is most convenient but it puts a little complexity into the code. 

Let's iterate the topic and maybe get other people's ideas. This issue should stay open for a while.

**Comment by am93** [comment]
Agreed. Right now I would prefer option 2. For first iteration I would recommend additional option when running application and show just parent directory name. In next iteration it could be improved with custom configuration file field.

If you need I can offer assistance in implementation.
